tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -827,6 +829,24 @@ public void testNoAcceptHeader() {
     }
 
 
+    @Test
+    @RunAsClient
+    @InSequence(41)
+    public void testCustomUnitAppendToGaugeName() {
+        Header wantPrometheusFormat = new Header("Accept", TEXT_PLAIN);
+        given().header(wantPrometheusFormat).get("/metrics/application").then().statusCode(200)
+        .and().body(containsString("TYPE application:org_eclipse_microprofile_metrics_test_metric_app_bean_gauge_me_b_hands gauge"));
+    }
+
+    @Test
+    @RunAsClient
+    @InSequence(42)
+    public void testCustomUnitForCounter() {
```

Comment:
rename to `testNoCustomUnitForCounter()`?

Reply:
SmallRye implementation includes the unit name in this case. Not sure if that is correct.
And also, you have specified `absolute=true` for this metric, so I'm getting this:
`TYPE application_metric_test_test1_count_me_b_jellybean counter`